pdns/lua-base4.cc

```diff
@@ -96,7 +96,6 @@ void BaseLua4::prepareContext() {
   d_lw->registerFunction<DNSName(DNSResourceRecord::*)()>("qname", [](DNSResourceRecord& rec) { return rec.qname; });
   d_lw->registerFunction<DNSName(DNSResourceRecord::*)()>("wildcardName", [](DNSResourceRecord& rec) { return rec.wildcardname; });
   d_lw->registerFunction<string(DNSResourceRecord::*)()>("content", [](DNSResourceRecord& rec) { return rec.content; });
-  d_lw->registerFunction<time_t(DNSResourceRecord::*)()>("lastModified", [](DNSResourceRecord& rec) { return rec.last_modified; });
```

Comment:
This is a documented Lua API function. Code might break if we remove it. Maybe just return 0?

Reply:
It is impossible to put a value in last_modified, so it is highly unlikely this function was ever used.

Reply:
If you say so... 
But at least the documentation should be adapted and an upgrade guide entry should be added.